src/Connectors/src/ConnectorCore/RabbitMQProviderServiceCollectionExtensions.cs

```diff
@@ -10,6 +10,7 @@
 using Steeltoe.Connector.Services;
 using System;
 using System.Linq;
+using System.Reflection;
```

Comment:
If you push more changes to other files before merging, I would back out changes to this file as there's no reason for there to be any changes in here